lib/tasks/generate.rake

```diff
@@ -0,0 +1,7 @@
+namespace :generate do
+  desc 'Generate fake data; Example: rake "generate:fake_data'
+  task fake_data: :environment do
```

Comment:
Proposed name change: `db:seed_users`.

Reply:
Changed it to `db:seed_users_data` instead of `seed_users` cuz that might make it sound like you're creating users.